x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_control.tsx

```diff
@@ -134,7 +134,7 @@ export class DrawControl extends Component<Props> {
         },
         paint: {
           'text-color': '#fbb03b',
-          'text-halo-color': 'rgba(255, 255, 255, 1)',
+          'text-halo-color': 'rgba(105, 112, 125, 1)',
```

Comment:
```suggestion
          'text-halo-color': 'rgba(0, 0, 0, 1)',
```


I've been running some color tests and the `rgba(105, 112, 125, 1)` doesn't provide enough contrast. So I suggest using a pure black as the `text-halo-color`.

<img width="780" alt="radius@2x" src="https://user-images.githubusercontent.com/2750668/124778255-c028e400-df38-11eb-836f-03b402ffe14b.png">